ZenPacks/zenoss/ZenPackLib/lib/tests/TestCase.py

```diff
@@ -37,7 +39,7 @@ def afterSetUp(self):
         try:
             zenpack_module = importlib.import_module(self.zenpack_module_name)
         except Exception:
-            LOG.exception("Unable to load zenpack named '%s' - is it installed? (%s)", self.zenpack_module_name)
+            self.LOG.exception("Unable to load zenpack named '%s' - is it installed? (%s)", self.zenpack_module_name)
```

Comment:
i think this needs another argument supplied to it or remove one of the formatters